index.js

```diff
@@ -79,6 +85,11 @@
   }
   extend(Just, Maybe);
 
+  //  Just#bind :: (a -> m b) -> m b
+  Just.prototype.bind = function(f) {
+    return f(this.value);
+  };
+
```

Comment:
this is the same as map, so you could use a reference to the same function if you were so inclined. not a big deal.